service/src/main/kotlin/fi/espoo/evaka/varda/VardaDevController.kt

```diff
@@ -0,0 +1,29 @@
+package fi.espoo.evaka.varda
+
+import fi.espoo.evaka.shared.db.Database
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.DeleteMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/varda-dev")
+class VardaDevController(
+    private val vardaUpdateService: VardaUpdateService
+) {
+    @DeleteMapping("/child/{vardaChildId}")
+    fun deleteChild(
+        db: Database.Connection,
+        @PathVariable vardaChildId: Long
+    ): ResponseEntity<Unit> {
+        if (System.getenv("VOLTTI_ENV") != "staging") {
+            return ResponseEntity.notFound().build()
+        }
+        vardaUpdateService.deleteFeeDataByChild(vardaChildId, db)
```

Comment:
Should we wrap all the deletes in a try catch and report any errors with info which child deletion failed (varda id, person id)? The situation requires manual work so it would be helpful to log necessary information for that. Of course an error code will also have to be returned to the caller so that the calling end will know that the child cannot be processed further.

Reply:
Added some logging, let's see if it suffices